env.template

```diff
@@ -1,8 +1,6 @@
 DOMAIN=
-TESTING=
+SPECSTORE=
+API=
 AUTH_TOKEN=
 AUTH_TOKEN_PERMISSION=
-OWNERID=
-DATAPACKAGE_NAME=
-REDIRECTION_DATASET=
-SPECSTORE_API=
\ No newline at end of file
+OWNERID=
```

Comment:
neither requested this

Reply:
I added, because it needs for running locally, otherwise it will fail